support/constants/constants.go

```diff
@@ -1,4 +1,4 @@
-package config
```

Comment:
any particular reason to move this file? To keep scope concise can we please keep it and only reconsider moving in a different PR?

Reply:
I encountered a circular dependency issue between support/util and support/config when attempting to reference the constants in the APIPortForLocalZone function. To resolve this, I moved the file.

Perhaps I could address the file relocation in a separate PR, which this current PR can then depend on ?